docs/docsite/rst/community/collection_contributors/collection_integration_updating.rst

```diff
@@ -131,7 +131,7 @@ We will add the following code to the file.
   - name: Test for new_option, create new user WITH the attribute
     postgresql_user:
```

Comment:
```suggestion
    community.postgresql.postgresql_user:
```

Reply:
Can you please change all occurrences by adding FQCN `community.postgresql.` to postgresql modules?

Reply:
@Akasurde I'd like to open a separate issue for this. Partly because this is a hacktoberfest issue so I'd prefer to keep the scope limited to boolean fixes as much as possible. And partly because if the problem is here, it might be in a few other of the community example files so I want to scan them all and then open a new issue to track fixing them all.